python/tank/platform/qt/login.ui

```diff
@@ -0,0 +1,228 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>LoginDialog</class>
+ <widget class="QDialog" name="LoginDialog">
+  <property name="windowModality">
+   <enum>Qt::ApplicationModal</enum>
```

Comment:
Yes, it’s probably not a good idea to have the window be modal in a DCC. I switched it to NonModal for now. We might want it to be modal relative to a Toolkit app window in the future, however because of threading it is hard to know which app the session renewal is coming from so we can’t put it modal to any specific app.